aws/data_source_aws_instance.go

```diff
@@ -208,6 +208,8 @@ func dataSourceAwsInstance() *schema.Resource {
 							Computed: true,
 						},
 
+						"tags": tagsSchema(),
```

Comment:
Should be `Computed: true` as its not expected in the configuration (this helper is Optional+Computed, but that's a later problem):

```suggestion
						"tags": tagsSchemaComputed(),
```